website/docs/components/link/standalone/partials/guidelines/guidelines.md

```diff
@@ -22,7 +22,7 @@ In most cases use a medium size, but use a link size that best fits the UI.
 
 For example:
 
-- Don't use large links in tables.
+- don’t use large links in tables.
```

Comment:
Probably best to address this in the future, but there are a bunch of instances of single bulleted lists which are a bit counter-intuitive.

Reply:
Let's chat in Slack about this, I was also curious about how we should handle this.